packages/flutter/lib/src/semantics/semantics_event.dart

```diff
@@ -88,3 +88,27 @@ class AnnounceSemanticsEvent extends SemanticsEvent {
     };
   }
 }
+
+/// An event which triggers long press semantic feedback.
+/// 
+/// Only used on Android.
```

Comment:
Maybe change to: "currently only honored on Android" and say a word about how android typically reacts to these events?

Reply:
Update doc on the semantics event themselves